src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedEventAccessorSymbol.cs

```diff
@@ -25,7 +25,7 @@ internal sealed class SynthesizedEventAccessorSymbol : SourceEventAccessorSymbol
         private readonly object _methodChecksLockObject = new object();
 
         internal SynthesizedEventAccessorSymbol(SourceEventSymbol @event, bool isAdder, EventSymbol explicitlyImplementedEventOpt = null, string aliasQualifierOpt = null)
-            : base(@event, null, @event.Location, explicitlyImplementedEventOpt, aliasQualifierOpt, isAdder, isIterator: false, isNullableAnalysisEnabled: false)
+            : base(@event, null, @event.Location, explicitlyImplementedEventOpt, aliasQualifierOpt, isAdder, isIterator: false, isNullableAnalysisEnabled: false, hasAnyBody: false, isExpressionBodied: false)
```

Comment:
>false [](http://example.com/codeflow?start=174&length=5)

This doesn't look like the value that was recommended in the comment for ```SourceEventAccessorSymbol```

Reply:
Not sure what you mean here.  Could you clarify?

Reply:
> Not sure what you mean here. Could you clarify?

Here is the comment that I am referring to https://github.com/dotnet/roslyn/pull/68158#discussion_r1191649763

Reply:
Got it.  thanks.

Reply:
Done.